healthlib/Plugin.cs

```diff
@@ -24,84 +24,61 @@ void Start()
             Utilla.Events.GameInitialized += OnGameInitialized;
         }
         
-        int getHealth()
-        {
-            return Health;
-        }
-
-        void OnEnable()
-        {
-            HarmonyPatches.ApplyHarmonyPatches();
-        }
+        int getHealth() => return Health;
 
-        void OnDisable()
-        {
-            HarmonyPatches.RemoveHarmonyPatches();
-        }
+        void OnEnable() => HarmonyPatches.ApplyHarmonyPatches();
+        void OnDisable() => HarmonyPatches.RemoveHarmonyPatches();
 
+        // BepInEx provides a logger property in the BaseUnityPlugin
+        // doing this automatically adds the plugin name and also appropriately represents severity
+        // https://docs.bepinex.dev/articles/dev_guide/plugin_tutorial/3_logging.html
         void OnGameInitialized(object sender, EventArgs e)
-        {
-            Debug.Log("HealthLib: Credits | Thanks for using HealthLib - pylua.code");
-        }
+            Logger.LogInfo("Thanks for using HealthLib - pylua.code");
 
         void OnTag(object sender, InfectionTagEventArgs e)
         {
             if (e.taggedPlayer.IsLocal && e.taggingPlayer != null)
-            {
                 removeHealth(10);
-            
-            }
         }
 
         void Update()
         {
             if (Health <= 0)
             {
                 Photon.Pun.PhotonNetwork.Disconnect();
-                Debug.Log("HealthLib: Action | Kicked from room");
+                Logger.LogInfo("Kicked from room");
             }
         }
 
         void addHealth(int healthadded)
         {
             if (Health < 100)
             {
-                Debug.Log("HealthLib: Action | Health Added: " + healthadded);
+                Logger.LogInfo($"Health added: {healthadded}");
```

Comment:
having the little dollar sign before the quotations in a string lets you do variables inline, by wrapping them in curly brackets. use two brackets instead of one to escape this: `$"This is my string with a variable of value {this}. But, I want curly brackets to actually show in the string, so do it like {{this!}} "`

Reply:
Thanks for letting me know al this going to use it in the future (also not going to reupload this mod, going to make a much cooler thing based around this)

Reply:
Still gonna test it cause the new mod would use this mod as a base